config/webpack.common.js

```diff
@@ -125,7 +125,7 @@ module.exports = function (options) {
        *
        * See: https://webpack.js.org/configuration/resolve/#resolve-modules
        */
-      // modules: [ path.resolve(__dirname, 'src'), path.resolve(__dirname, 'node_modules')]
+      modules: [ helpers.root('src'), helpers.root('node_modules')]
```

Comment:
You mean that the configuration itself option is not allowed? Double check this one, as this solved the link problem for me.